gen3-client/g3cmd/configure.go

```diff
@@ -23,14 +23,15 @@ func init() {
 	var configureCmd = &cobra.Command{
 		Use:   "configure",
 		Short: "Add or modify a configuration profile to your config file",
-		Long: `Configuration file located at ~/.gen3/config
+		Long: `Configuration file located at ~/.gen3/gen3_client_config.ini
```

Comment:
yeah that is unfortunate ☹️ But I think we have to move forward with the `go.ini` solution, since the way we are storing and parsing configs was very unreliable (assuming profiles are simply separated by a certain number of lines)

underthe hood the file content deos not have a very huge difference, the newer config will have better formating, I can also keep the file name, but that will cause user's old config file be overwritten

Reply:
nah, i like the new file name, `config` was too vague 😄